src/args.ts

```diff
@@ -0,0 +1,49 @@
+interface iCommands {
+  command: string;
+  description: string;
+}
+
+const commands: iCommands[] = [
+  {
+    command: "--help",
+    description: "Show CLI usage.",
+  },
+  {
+    command: "--version\t",
```

Comment:
Curious about this `\t`. Is it for the output?

Reply:
yes it's used to keep all descriptions at the same "column" 

**with the \t**
```
 --version                      Print prettierd version.
 --ignore-path <path>           Path to a file with patterns describing files to ignore.
```

**without the \t**
```
 --version  Print prettierd version.
 --ignore-path <path>  Path to a file with patterns describing files to ignore.```

Reply:
Gotcha. Ideally this is something we'd handle when displaying the help, but we can ship as is and improve later. Can you include a comment about that?